.github/workflows/ci.yml

```diff
@@ -78,6 +79,7 @@ jobs:
         namespace: ["default","flink"]
         mode: ["native", "standalone"]
         java-version: [ 11, 17, 21 ]
+        http-client: [ "okhttp", "vertx", "jetty",  "jdk" ]
```

Comment:
I suggest to only test the different HTTP clients in a single test case such as:
`v1_20, default, native, java21 and test_application_operations.sh `

we don't want to blow up our matrix by a factor of 4 :D

Reply:
I think https://github.com/apache/flink-kubernetes-operator/pull/881/commits/7b74210e8d64acb881a268ac8f37271de155c060 does the job

Reply:
I've changed tack and created a smoke test job in https://github.com/apache/flink-kubernetes-operator/pull/881/commits/28535546dbf887d7bd85f7f13dce1211540c27dc.